server.js

```diff
@@ -4,6 +4,9 @@ const app = require('./src/app')
 const config = require('./src/config')
 const PORT = config('PORT')
 
+const database = require('./databaseConnection')
+const sampleModel = require('./src/sampleModel')
+
 app.listen(PORT, function () {
```

Comment:
Let's call the function that connects to the db in the callback after the server is started

Reply:
Fixed it.